### ISSUE TITLE
Re-evaluate the Os and Arch filtering

### ISSUE THREAD

**Issue by mattfarina** [Masterminds/glide#570]
The way Glide has changed these may no longer be needed.

**Comment by sdboyer** [comment]
Yeah, as they exist now, I think they can be safely removed. I don't see a need for them in the manifest at all, probably even in the long term.

I _do_ see a case for them in the lock file, though gps (at least) doesn't support that planned behavior yet. (I've removed os and arch in #384 in both manifest and lock, though we could always put them back in)
